osdep/mpv.exe.manifest

```diff
@@ -10,6 +10,7 @@
     <application xmlns="urn:schemas-microsoft-com:asm.v3">
         <windowsSettings xmlns:ws="http://schemas.microsoft.com/SMI/2005/WindowsSettings">
             <ws:dpiAware>True/PM</ws:dpiAware>
+            <dpiAwareness xmlns="http://schemas.microsoft.com/SMI/2016/WindowsSettings">PerMonitorV2,PerMonitor</dpiAwareness>
```

Comment:
It's kind of weird to use a namespace prefix for ``<ws:dpiAware>`` and the default namespace for ``<dpiAwareness>``. ``<ws:dpiAware>`` should probably be changed to use ``<dpiAware xmlns="...`` to match the following line instead.